olive/passes/onnx/__init__.py

```diff
@@ -31,7 +30,6 @@
     "AppendPrePostProcessingOps",
     "DynamicToFixedShape",
     "GenAIModelExporter",
-    "GptqQuantizer",
```

Comment:
Please also update the docs to move it from `sources/features/passes/onnx` to `pytorch`.